public/theme/boltcms-2021/css/app.css

```diff
@@ -1877,4 +1878,71 @@ header > nav > ul > li > a {
     display: flex;
     text-decoration: none;
     transition: all linear 0.1s;
+}
+
+
+/** Bolt form */
```

Comment:
@mcdennem I think @Dsduit already created some styles for the forms, which are pushed live, see: https://boltcms.io/submit-showcase

So, I don't know if we need that..

Reply:
Correct, this file is not used anymore, so I will ⚡ it